src/main/java/roboy/memory/nodes/Roboy.java

```diff
@@ -0,0 +1,132 @@
+package roboy.memory.nodes;
+
+import roboy.dialog.Config;
+import roboy.memory.Neo4jMemory;
+import roboy.memory.Neo4jRelationships;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Encapsulates a MemoryNodeModel and enables dialog states to easily store
+ * and retrieve information about Roboy.
+ */
+public class Roboy {
+    private MemoryNodeModel roboy;
+    Neo4jMemory memory;
+    // Memory is not queried in NOROS mode.
+    private boolean memoryROS;
```

Comment:
?

Reply:
This checks are provided through the Config class as a flag to let the nodes know if they are supposed to access/flush the data to the Memory or leave it intact for "no memory" configuration testing purposes (that was done before me)